blockchain/votebits_test.go

```diff
@@ -1116,26 +1171,86 @@ func TestVoting(t *testing.T) {
 				},
 			},
 		},
-		//{
-		//	name:              "less than quorum",
-		//	numNodes:          params.RuleChangeActivationInterval,
-		//	vote:              pedro,
-		//	blockVersion: powVersion,
-		//	startStakeVersion: posVersion,
-		//	voteBitsCounts:    []voteCount{{vote: 0x02, count: 100}},
-		//	expectedState:     ThresholdStateTuple{ThresholdStarted, invalidChoice},
-		//},
+		{
+			name:              "pedro overlap",
+			vote:              pedro,
+			blockVersion:      powVersion,
+			startStakeVersion: posVersion - 1,
+			voteBitsCounts: []voteCount{
+				{
+					vote: VoteVersionTuple{
+						Version: posVersion - 1,
+						Bits:    0x01},
+					count: uint32(params.StakeValidationHeight) +
+						uint32(19*params.StakeVersionInterval) - 1,
+				},
+				{
+					vote: VoteVersionTuple{
+						Version: posVersion,
+						Bits:    0x01},
+					// This puts us at the first overlapped svi and rci
+					count: uint32(params.StakeVersionInterval) - 1,
+				},
+				{
+					vote: VoteVersionTuple{
+						Version: posVersion,
+						Bits:    0x01},
+					count: 1,
+				},
+				{
+					vote: VoteVersionTuple{
+						Version: posVersion,
+						Bits:    0x03},
+					count: uint32(params.RuleChangeActivationInterval) - 1,
+					// This puts us at the next rci
```

Comment:
Same here.  This is one block before the rci, the next one is what reaches the rci.